lang/en.yml

```diff
@@ -341,16 +381,17 @@ en:
     EditLink: edit
     HEADER: 'This is a virtual page'
     HEADERWITHLINK: 'This is a virtual page copying content from "{title}" ({link})'
+    OverrideNote: 'Overrides inherited value from the source'
     PLURALNAME: 'Virtual Pages'
     PLURALS:
       one: 'A Virtual Page'
       other: '{count} Virtual Pages'
     PageTypNotAllowedOnRoot: 'Original page type "{type}" is not allowed on the root level for this virtual page'
     SINGULARNAME: 'Virtual Page'
-    db_CustomMetaDescription: 'Custom Meta Description'
-    db_CustomExtraMeta: 'Custom Meta Tags'
+    db_CustomExtraMeta: 'Custom extra meta'
```

Comment:
Why are these changing?

Reply:
https://github.com/silverstripe/silverstripe-cms/pull/3080#discussion_r2094630679

Reply:
Leave as is per https://github.com/silverstripe/silverstripe-cms/pull/3080/files#r2094642942